src/classlibnative/bcltype/objectnative.cpp

```diff
@@ -183,7 +183,7 @@ FCIMPL2(FC_BOOL_RET, ObjectNative::Equals, Object *pThisRef, Object *pCompareRef
     if (pThisMT != pCompareRef->GetMethodTable())
         FC_RETURN_BOOL(FALSE);
 
-    // Compare the contents (size - vtable - sink block index).
+    // Compare the contents (size - vtable - sync block index).
```

Comment:
I was looking at `ComEventsSink`, `StackBuilderSink` classes and 41 other occurrences of Sink in CoreCLR's C++ code. If you are absolutely sure, then sync that is, but please confirm.